### ISSUE TITLE
Music looping has a gap between the loops

### ISSUE THREAD

**Issue by hakanai** [corbindavenport/wii-shop-extension#69]
If you listen to the music as it hits the end of the track and loops over, there is a gap between the two plays.

I checked the .ogg file itself, and the file loops properly in a normal media player, so I guess the problem is somewhere in the code?

**Comment by hakanai** [comment]
After digging through the code a bit, the only thing I see is setting `loop` to `true` on the `Audio` object. So this is a Chrome bug? Gross... weird how I haven't noticed it with any other looping audio on web sites and the like.

**Comment by corbindavenport** [comment]
Yeah, I'm not sure if HTML5 audio objects have any kind of delay/crossfade options. I might have to switch to a different audio playback method anyway for #7, which might have better options.

**Comment by hakanai** [comment]
Web Audio API apparently does allow for this because people are even using it for drum loops.

I just don't understand why Google didn't then make Audio work on top of the newer API so that it would properly loop. Maybe they are, and it just isn't finished yet.
